src/tools/lifter/models/l3mod/m0_mod_stepLib.sml

```diff
@@ -0,0 +1,129 @@
+structure m0_mod_stepLib =
+struct
+
+  open HolKernel Parse boolLib bossLib;
+
+  open m0_mod_stepTheory;
+
+(*
+open m0Theory m0_stepTheory;
```

Comment:
Those comments make me really sad (because that's everywhere). You basically have tests, but you don't use them. It's easy to turn that into a test function! That way, you can track regressions early on :wink:

Reply:
yes, absolutely right. will do